index.html

```diff
@@ -0,0 +1,349 @@
+            </p>
+          </div>             
+          <div class="lets-connect lets-connect-about">
+            <p class="lets-connect-statement">Let's connect</p>
+  
+            <ul class="social-networks about-myself-social-ntks">
+              <li>
+              <a href="#" class="social-network">
+                <i class="fa fa-twitter"></i>
+              </a>
+              </li>                        
+              <li>
+              <a href="#" class="social-network">
+                <i class="fa fa-linkedin"></i>
+              </a>
+              </li>
+              <li>
+              <a href="#" class="social-network">
+                <i class="fa fa-medium"></i>
+              </a>
+              </li>
+              <li>
+              <a href="#" class="social-network">
+                <i class="fa fa-github"></i>
+              </a>
+              </li>
+              <li>
+              <a href="#" class="social-network">
+                              <i class="fa fa-angellist"></i>
+              </a>
+              </li>
+              </ul>                
+          </div>
+          <div class="action">
+            <button type="button" class="btn about-me-btn">Get my resume </button>
+          </div>
+        </div>
+        <ul class="skill-list">
+            <!----<div class="frame-12"> --->
+            <li class="frame-7-languages">
+              <a href="#">Languages</a>
+              <i class='fa fa-chevron-down'></i>
+
+               <!---<ul class="language-list"> ---> 
+              <li>
+                <ul class="language-list">               
+                  <li class="frame-9-10-11">                           
+                      <img src="./images/Ellipse-1.png" alt="Ellipse 1(Javascript)">
+                      <a href="#">Javascript</a>                
+                  </li>   
+                  <li class="frame-9-10-11">                    
+                      <img src="./images/ellipse-2.png" alt="Ellipse 1(HTML)">
+                      <a href="#">HTML</a>                             
+                  </li>                         
+                  <li class="frame-9-10-11">               
+                      <img src="./images/ellipse-3.png" alt="Ellipse 3(CSS)">
+                      <a href="#">CSS</a>                    
+                  </li>
+                </ul>
+              </li>
+            
+            <li class="skill-and-framework">
+              <a href="#">Frameworks</a>
+              <i class='fa fa-chevron-right'></i>
+            </li>
+            <li class="skill-and-framework">
+              <a href="#">Skills</a>
+              <i class='fa fa-chevron-right'></i> 
+            </li>
+        </ul>
+      </div>           
+
+    </section>
+    <!---------End of about myself section-------->
+    <!---------End of mobile-finish-version----->
+
+
+    <!--------Form container mobile version------>
+    <footer>
```

Comment:
- [ ] Good job with your contact section design  👍🏽 . I think it will be better if your UI looks like the one on the Figma design template as I can see that your background doesn't really match the Figma design. Please check it out carefully  😄 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/187533216-d5e37768-0508-44df-9d49-f39b908d26d3.png)  | ![image](https://user-images.githubusercontent.com/63473584/187533026-d853685f-5c4f-4ec9-917c-bc63c5d3a5c6.png)  |

**Suggested clue:** I think the background for the contact form overlaps the white background for the about section 😉